packages/eslint-plugin/src/rules/pkg-renaming.ts

```diff
@@ -0,0 +1,88 @@
+import { AST_NODE_TYPES } from '@typescript-eslint/experimental-utils'
+import { createRule } from '../utils'
+
+const simpleMappings = new Map<unknown, string>([
```

Comment:
Because the type of the `source` property on `ImportDeclaration` nodes is `Literal`, which can be more than just `string` so we need to allow all of those type of values to be passed:

![image](https://user-images.githubusercontent.com/3151613/154822362-c4e1324b-4e3d-4df8-8eea-e0b582632222.png)